brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Optimize timeline js

### Description

- Remove useless variable and use a switch condition instead of an if-else condition.
- Trello - https://trello.com/c/aYwSrZ1f/156-googles-web-vitals-performance-issue

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Content Timeline and Post Timeline.
- Check layout setting ( Orientation, Arrow alignment, Stack-on .etc )

### Checklist:
